app/views/rockles/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<ul>
+
+  <% @rockles.each do |rockle| %>
+
+<li><%= rockle.name %>:
+  <br/>
+  <span>Owner: <a href="<%= user_url({id: rockle.user.id}) %>"><%= rockle.user.username %></a></span>
```

Comment:
this works, but rails is fancy and you can just pass in the user object: user_url(rockle.user)